container-chains/templates/frontier/node/src/chain_spec.rs

```diff
@@ -202,7 +193,7 @@ pub fn local_testnet_config(para_id: ParaId) -> ChainSpec {
 }
 
 fn testnet_genesis(
-    invulnerables: Vec<(AccountId, NimbusId)>,
+    _invulnerables: Vec<(AccountId, NimbusId)>,
```

Comment:
I'd say lets remove this argument if not used, in both frontier and simple chain spec

Reply:
Yeah just wanted to see it compile. Will do :+1:

Reply:
@nbaztec I think you only needed to remove the `_invulnerables ` parameter